lib/src/test/java/io/ably/lib/test/realtime/RealtimeChannelTest.java

```diff
@@ -3,6 +3,7 @@
 import io.ably.lib.debug.DebugOptions;
 import io.ably.lib.realtime.*;
```

Comment:
I realise this code existed before this PR, but these are inconsistent import styles. I think generally we're avoiding wildcard import statements.

Reply:
Thanks for pointing it out, fixed.

Reply:
I thought we had this covered by checkstyle... but perhaps that doesn't run over the test classes. 🤔